docs/UserGuide.md

```diff
@@ -389,15 +392,15 @@ For example:
-| [**Add**](#adding-a-contact-add)    | `add n/NAME e/EMAIL r/ROLE a/ADDRESS c/COURSE [t/TAG]… [p/PHONE]` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com r/STUDENT a/PGPR c/CS2103T t/friend` |
-| [**Clear**](#clearing-all-entries-clear)  | `clear`                                                                                                                                                                         |
-| [**Delete**](#deleting-a-contact-delete) | `delete INDEX`<br> e.g., `delete 3`                                                                                                                                             |
-| [**Edit**](#editing-a-contact-edit)   | `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [r/ROLE] [a/ADDRESS] [c/COURSE] [t/TAG]…`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`                                        |
-| [**Find**](#locating-contacts-find)   | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake CS2103T STUDENT`                                                                                                      |
-| [**List**](#listing-all-contacts-list)   | `list`                                                                                                                                                                          |
-| [**Help**](#viewing-help-help)   | `help`                                                                                                                                                                          |
-| [**Exit**](#exiting-the-program-exit)   | `exit`                                                                                                                                                                          |
-| [**Copy Email**](#copying-a-contacts-email-to-clipboard-copy)   | `copy INDEX`                                                                                                                                                                          |
-| [**Undo**](#undoing-the-last-command-undo)   | `undo`                                                                                                                                                                          |
+| Action     | Format, Examples                                                                                                                                                          |
+|------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [**Add**](#adding-a-contact-add)    | `add n/NAME e/EMAIL r/ROLE c/COURSE [a/ ADDRESS] [p/PHONE] [t/TAG]…[f/]` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com r/STUDENT a/PGPR c/CS2103T t/friend` |
```

Comment:
Should we include "[f/]" here also? It is not a section like name and email and it is just kind of additional command. How do you think?

Reply:
Oh nevermind, we mentioned that it is also a field.